.github/workflows/release.yml

```
@@ -0,0 +1,40 @@
+        dotnet publish -c Release -r win-x64 --self-contained true /p:PublishSingleFile=true
+        
+    - name: Read Release Notes
+      id: release_notes
+      shell: pwsh
+      run: |
+        $notes = Get-Content "RELEASE_NOTES.md" -Raw
+        $notes = $notes -replace '\r\n', '%0A'
+        echo "::set-output name=content::$notes"
+        
+    - name: Create Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)